README.md

```diff
@@ -1,10 +1,12 @@
 [![Build Status][build-badge]][build-status]
 [![Coverage Status][coverage-badge]][coverage-status]
 
-# Ecosystem Identifier (EIDER) Service
+# Ecosystem Identifier Service
 
 <a name="readme-top"></a>
 
+A Spring backend for the *Ecosystem Identifier (EIDER) Service* developed and maintained by [Texas A&M University Libraries][tamu-library].
+
 This service provides ecosystem identifier generation.
```

Comment:
Perhaps "This service mints identifiers for entities in the digital asset management ecosystem.  It also affords registration of alternate identifiers and provides redirection from alternate identifiers to entities."